rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -308,31 +330,31 @@ namespace rocal{
     PYBIND11_MODULE(rocal_pybind, m) {
         m.doc() = "Python bindings for the C++ portions of ROCAL";
         // rocal_api.h
-        m.def("rocalCreate",&rocalCreate,"Creates context with the arguments sent and returns it",
+        m.def("rocalCreate", &rocalCreate,"Creates context with the arguments sent and returns it",
                 py::return_value_policy::reference,
                 py::arg("batch_size"),
                 py::arg("affinity"),
                 py::arg("gpu_id") = 0,
                 py::arg("cpu_thread_count") = 1,
                 py::arg("prefetch_queue_depth") = 3,
                 py::arg("output_data_type") = 0);
-        m.def("rocalVerify",&rocalVerify);
-        m.def("rocalRun",&rocalRun);
-        m.def("rocalRelease",&rocalRelease);
+        m.def("rocalVerify", &rocalVerify);
+        m.def("rocalRun", &rocalRun, py::return_value_policy::reference);
```

Comment:
py::return_value_policy::reference is added for the pybind for functions which returns pointers or other objects which needs to be freed from C++. rocalVerify just returns a enum type so it can be deleted from python itself